code/game/machinery/ARES/_ARES_defines.dm

```diff
@@ -0,0 +1,56 @@
+/// Generic access for 1:1 conversations with ARES and unrestricted commands.
+#define ARES_ACCESS_BASIC 0
+/// Secure Access, can read ARES Announcements and Bioscans.
+#define ARES_ACCESS_COMMAND 1
+#define ARES_ACCESS_JOE 2
+/// CL, can read Apollo Log and also Delete Announcements.
+#define ARES_ACCESS_CORPORATE 3
+/// Senior Command, can Delete Bioscans.
+#define ARES_ACCESS_SENIOR 4
+/// Synth, CE & Commanding Officer, can read the access log.
+#define ARES_ACCESS_CE 5
+#define ARES_ACCESS_SYNTH 6
+#define ARES_ACCESS_CO 7
+/// High Command, can read the deletion log.
+#define ARES_ACCESS_HIGH 8
+#define ARES_ACCESS_WY_COMMAND 9
+/// Debugging. Allows me to view everything without using a high command rank. Unlikely to stay in a full merge.
+#define ARES_ACCESS_DEBUG 10
+
+#define ARES_RECORD_ANNOUNCE "Announcement Record"
+#define ARES_RECORD_BIOSCAN "Bioscan Record"
+#define ARES_RECORD_BOMB "Orbital Bombardment Record"
+#define ARES_RECORD_DELETED "Deleted Record"
+
+/datum/ares_record
+	var/record_name = "ARES Data Core"
+	var/time
+	var/title
+	var/details
+	var/user
```

Comment:
Document these variables. Explain what format they're going to be in (e.g. time suggests it's going to be a number value but it looks like it's actually a string value)